pkg/bpf/map.go

```diff
@@ -216,6 +217,57 @@ func (m *Map) setPathIfUnset() error {
 	return nil
 }
 
+func (m *Map) migrate(fd int) (bool, error) {
```

Comment:
I'm just writing this as a precaution: between line 220 and 306 are any garbage of maps file descriptors left open?